hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieFileGroup.java

```diff
@@ -91,19 +91,19 @@ public void addDataFile(HoodieDataFile dataFile) {
    */
   public void addLogFile(HoodieLogFile logFile) {
     if (!fileSlices.containsKey(logFile.getBaseCommitTime())) {
-      fileSlices.put(logFile.getBaseCommitTime(), new FileSlice(logFile.getBaseCommitTime(), id));
+      fileSlices.put(logFile.getBaseCommitTime(), new FileSlice(partitionPath, logFile.getBaseCommitTime(), id));
     }
     fileSlices.get(logFile.getBaseCommitTime()).addLogFile(logFile);
   }
 
-  public String getId() {
-    return id;
-  }
-
   public String getPartitionPath() {
     return partitionPath;
   }
 
+  public HoodieFileGroupId getFileGroupId() {
```

Comment:
why is `HoodieFileGroupId` not a member on this class? Same q for FileSlice.

Reply:
+1

Reply:
Fixed.